extensions/machine-learning-services/src/modelManagement/registeredModelService.ts

```diff
@@ -56,22 +95,73 @@ export class RegisteredModelService {
 
 	private async runRegisteredModelsListQuery(connection: azdata.connection.ConnectionProfile): Promise<azdata.SimpleExecuteResult | undefined> {
 		try {
-			return await this._queryRunner.runQuery(connection, this.registeredModelsQuery(this._config.registeredModelDatabaseName, this._config.registeredModelTableName));
+			return await this._queryRunner.runQuery(connection, this.registeredModelsQuery(connection.databaseName, this._config.registeredModelDatabaseName, this._config.registeredModelTableName));
+		} catch {
+			return undefined;
```

Comment:
Not going to at least log something to console if an error occurs?

Reply:
the error is in the console, for now ignoring 

---
In reply to: [385788402](https://github.com/microsoft/azuredatastudio/pull/9377#discussion_r385788402) [](ancestors = 385788402)